tests/house_tests.py

```diff
@@ -108,6 +108,8 @@ def test_profession_either_parent(self):
         house2.save()
         houses = _filter_houses_by_form(form_data_filter_tests(), House.objects.all())
         assert len(houses) == 2
+        assert House.objects.get(parent_profession_1=Job.TEACHER) in houses
+        assert House.objects.get(parent_profession_2=Job.TEACHER) in houses
```

Comment:
Why is this here?

Reply:
I accidentally deleted these lines while performing a rebase, so I returned them